### DISCUSSION TITLE
订阅规则

### DISCUSSION THREAD

**Discussion by xxtyyq** [jxxghp/MoviePilot#3206] [问题咨询]
请教各位大神，我在订阅规则的“包含”项里输入了“中字,free”，中字没问题，但是free没有起作用，依然会下到50%等资源，换了中文的“免费”也不行，我应该如何设置才能自动判定是不是免费资源？

**Comment by cng10001** [comment]
促销或者免费跟你手输`free`没关系。
看下文档里`订阅模式`的区别。https://wiki.movie-pilot.org/zh/subscribe
然后勾选软件本身提供的免费规则选项
![image](https://github.com/user-attachments/assets/a10f7d7e-988d-418d-ba91-ad804945fa4e)

**Reply by xxtyyq** [reply]
明白了，谢谢！
